transcom/mymove
# B 19982-2-INT-KB

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A949550)
## [OG INT PR](https://github.com/transcom/mymove/pull/12613)
## [INT KB 1](https://github.com/transcom/mymove/pull/12899)

## Summary

> [!IMPORTANT]  
> Needs changes from 19988 to work

Update the services related to PPM moving expense document submission to update the appropriate "submitted_" database column for additional data that is adjustable by closeout counselors.

How to test
As a customer, submit PPM documents for a moving expense.
Confirm your data is stored in the related "submitted_" column of the appropriate table
